notebooks/MAST/Identifying-transient-candidates/identifying_highz_candidates.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>worth commenting after this cell that even finding a galaxy @Z=6 seems to exceed Hubble's normal capabilities?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>